simpleclient/src/main/java/io/prometheus/client/Collector.java

```diff
@@ -91,8 +97,18 @@ public boolean equals(Object obj) {
           return false;
         }
         Sample other = (Sample) obj;
+
+        boolean timestampEquals = true;
```

Comment:
This is a bit long, I think you can simplify a bit.

Reply:
Done. Now has been reduced to one line.